api/control/v1alpha1/types.go

```diff
@@ -589,6 +601,25 @@ const (
 	TypeArray  ValueType = "Array"
 )
 
+// RowEvaluationError reports an error that is specific to a WEC
+// and a column.
+type RowEvaluationError struct {
+	WEC Destination `json:"wec"`
+	// ColumnName is the empty string for the filter expression
+	ColumnName string `json:"columnName"`
+	Error      string `json:"error"`
+}
+
+// FilterColumnName is the ColumnName value used to report an evaluation error
+// in the filter expression of a StatusCollector.
+const FilterColumnName = ""
```

Comment:
Yes, this is trying to avoid taking a noticeable bit out of the space of column names.
We do not currently define any restriction on column names. That is probably something worth changing in its own right. I wonder what would be a reasonable space to allow for column names? For one thing, I would probably restrict it to be printable (non-whitespace) [Basic Latin](https://en.wikipedia.org/wiki/Basic_Latin_(Unicode_block)) characters. i wonder what other restrictions would be reasonable. Or perhaps we should start with something very restricted, such as allowing only a domain name label, and relax it if/when the need arises.

Reply:
I think it might make sense to reserve a prefix such as "KS_" or anything similar. In all cases - not a blocker for this PR.